compiler/parser/RdrHsSyn.hs

```diff
@@ -2779,6 +2779,7 @@ mkTemplateInstanceDecl sharedBinds templateName conName ValidTemplate{..} =
   , mkInstance "HasToAnyTemplate" $ mkPrimMethod "_toAnyTemplate" "EToAnyTemplate"
   , mkInstance "HasFromAnyTemplate" $ mkPrimMethod "_fromAnyTemplate" "EFromAnyTemplate"
   , mkInstance "HasTemplateTypeRep" $ mkPrimMethod "_templateTypeRep" "ETemplateTypeRep"
+  , mkInstance "HasIsInterfaceType" $ mkTemplateClassMethod "_isInterfaceType" [proxy] (mkQualVar $ mkDataOcc "False") Nothing
```

Comment:
should we add that typeclass to our dummy stdlib for testing?

Reply:
Done!